pkg/router/template/router_test.go

```diff
@@ -360,6 +361,9 @@ func TestCreateServiceAliasConfig(t *testing.T) {
 	serviceName := "TestService"
 	serviceWeight := int32(0)
 
+	var headerNameXFrame string = "X-Frame-Options"
+	var headerNameXSS string = "X-XSS-Protection"
+
```

Comment:
These tests should be in their own function, `TestCustomHeaders` or something like that.

Reply:
Never mind, I see that you altered the `createServiceAlias` function to do this.  Hm.